### PR TITLE
removed all envoy proxy v1 code references

### PR DESCRIPTION
**Pull request by GregHanson**
does not remove the envoy/v1 folder to simplify code diff.  Folder will be removed in followup PR

This PR also does not address any updates to watcher.go and discovery.go for envoy v2 API

### CODE REVIEW

File: pilot/pkg/networking/plugin/mixer/filter.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Istio Authors
```
Comment (kyessenov) [inline pilot/pkg/networking/plugin/mixer/filter.go]: Remove this.

Comment (GregHanson) [reply pilot/pkg/networking/plugin/mixer/filter.go]: @kyessenov the entire file?

Comment (kyessenov) [reply pilot/pkg/networking/plugin/mixer/filter.go]: Yes.

Comment (GregHanson) [reply pilot/pkg/networking/plugin/mixer/filter.go]: @kyessenov done

File: pilot/pkg/networking/plugin/mixer/mixer.go [hunk 2]
```diff
@@ -351,4 +351,4 @@ func attrsCopy(attrs attributes) attributes {
 		out[k] = v
 	}
 	return out
-}
+}
```
Comment (kyessenov) [inline pilot/pkg/networking/plugin/mixer/mixer.go]: seems unnecessary to remove end of file.

File: pilot/pkg/proxy/envoy/cert-chain.pem [hunk 3]
```diff
@@ -0,0 +1 @@
+cert-chain.pem
```
Comment (rshriram) [inline pilot/pkg/proxy/envoy/cert-chain.pem]: ?

Comment (GregHanson) [reply pilot/pkg/proxy/envoy/cert-chain.pem]: not sure where it came from - removing

File: pilot/pkg/proxy/envoy/infra_auth_test.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 Istio Authors
+//
```
Comment (rshriram) [inline pilot/pkg/proxy/envoy/infra_auth_test.go]: remove this file.

File: pilot/pkg/proxy/envoy/infra_auth.go [hunk 5]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
+//
```
Comment (rshriram) [inline pilot/pkg/proxy/envoy/infra_auth.go]: and this file

**Pull request merged**